apps/src/javalab/Backpack.jsx

```diff
@@ -21,11 +21,12 @@ const Dialog = makeEnum('IMPORT_WARNING', 'IMPORT_ERROR');
 class Backpack extends Component {
   static propTypes = {
     displayTheme: PropTypes.oneOf(Object.values(DisplayTheme)).isRequired,
-    isDisabled: PropTypes.bool.isRequired,
+    isButtonDisabled: PropTypes.bool.isRequired,
```

Comment:
Note: I changed this prop name to avoid confusion with `backpackEnabled`. Its functionality is the same